tests/bot/exts/filters/test_token_remover.py

```diff
@@ -289,28 +294,52 @@ def test_format_log_message(self, log_message):
             hmac="x" * len(token.hmac),
         )
 
-    @autospec("bot.exts.filters.token_remover", "DECODED_LOG_MESSAGE")
-    def test_format_userid_log_message_bot(self, decoded_log_message):
+    @autospec("bot.exts.filters.token_remover", "UNKNOWN_USER_LOG_MESSAGE")
+    def test_format_userid_log_message_unknown(self, unknown_user_log_message):
         """
         Should correctly format the user ID portion of the log message when the user ID is
          not found in the server.
         """
         token = Token("NDcyMjY1OTQzMDYyNDEzMzMy", "XsySD_", "s45jqDV_Iisn-symw0yDRrk_jf4")
-        decoded_log_message.format.return_value = " Partner"
+        unknown_user_log_message.format.return_value = " Partner"
         msg = MockMessage(id=555, content="hello world")
         msg.guild.get_member = MagicMock(return_value=None)
 
         return_value = TokenRemover.format_userid_log_message(msg, token)
 
         self.assertEqual(
             return_value,
-            (decoded_log_message.format.return_value, False),
+            (unknown_user_log_message.format.return_value, False),
+        )
+        unknown_user_log_message.format.assert_called_once_with(
+            user_id=472265943062413332,
         )
-        decoded_log_message.format.assert_called_once_with(
+
+    @autospec("bot.exts.filters.token_remover", "KNOWN_USER_LOG_MESSAGE")
+    def test_format_userid_log_message_bot(self, known_user_log_message):
+        """
+        Should correctly format the user ID portion of the log message when the user ID is
+         not found in the server.
+        """
+        token = Token("NDcyMjY1OTQzMDYyNDEzMzMy", "XsySD_", "s45jqDV_Iisn-symw0yDRrk_jf4")
+        known_user_log_message.format.return_value = " Partner"
+        msg = MockMessage(id=555, content="hello world")
+        msg.guild.get_member = MagicMock(return_value=MagicMock(__str__=MagicMock(return_value="Sam"), bot=True))
```

Comment:
Since it's already a mock, there are two simpler ways to go about this. Not sure which I prefer.

```py
        msg.guild.get_member.return_value = MagicMock(__str__=MagicMock(return_value="Sam"), bot=True)
```

```py
        msg.guild.get_member.return_value.__str__.return_value = "Sam"
        msg.guild.get_member.return_value.bot = True
```

Reply:
I prefer the second, and since it seems alright I'll go with it